.github/workflows/firebase-hosting-pull-request.yml

```
@@ -34,7 +44,18 @@ jobs:
 
-      - uses: FirebaseExtended/action-hosting-deploy@v0
+      - name: 📈 Inject and upload PostHog source maps
+        run: |
+          pnpm install -g @posthog/cli
+          posthog-cli login
+          posthog-cli --host https://eu.posthog.com sourcemap inject --directory ./dist/assets
+          posthog-cli --host https://eu.posthog.com sourcemap upload --directory --delete-after ./dist/assets
+        env:
+          POSTHOG_CLI_ENV_ID: ${{secrets.POSTHOG_CLI_ENV_ID}}
+          POSTHOG_CLI_TOKEN: ${{secrets.POSTHOG_CLI_TOKEN}}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)